tests/app/test_jinja_filters.py

```diff
@@ -237,7 +237,7 @@ def test_format_duration(duration, formatted_duration, app):
 @pytest.mark.parametrize(
     "answer, width",
     (
-        ({}, 10),
+        ({}, 15),
```

Comment:
at the moment, if I allow decimal places, I can enter a 15 digit number _and_ two decimal places as the limit, should that be tested if thats correct?

Reply:
Sorry forgot to update the PR. It should say we should allow 15 whole intergers + up to 6 decimals. Will update PR

Reply:
thanks Yuyu, just looking at the tests here, I think it'd be good to have the extremes added to `test_get_width_for_number`

Reply:
Sorry for making a meal of the above comment, moving it here - but looking closer at this test, I'm not sure if its working exactly as intended

in the test_numbers json the "value" for the minimum and maximum is provided as an "identifier" and "source" rather than a number, but the function `get_width_for_number` is doing

`min_value = answer.get("minimum", {}).get("value", 0)`
`min_value_width=len(str(min_value))` 

which is going to use the length of the object as a string rather than the value, does this need updating? and should allowable_widths also be updated given the new limits. That said I'm not entirely sure how this is being used

Reply:
Re: `get_width_for_number`, good spot. But given that is an existing issue, I'll raise a card to look at that separately. Must have been an oversight when value sources for that were added.

Reply:
Added extremities 👍